yksgi/src/lib.rs

```diff
@@ -149,6 +168,271 @@ impl SGInterp {
         }
     }
 
+    // Generate frames from stackmap information after a guard failure. The new frames are stored
+    // inside some allocated memory whose pointer this function returns. The frames are then later
+    // `memcpy`ed to the actual stack by ykcapi::__ykrt_reconstruct_frames.
```

Comment:
another nitpick: backtick `ykcapi::__ykrt_reconstruct_frames`

Reply:
These should also be `///` doc string quotes. And then `[ykcapi::__ykrt_reconstruct_frames]` (i.e. put it in backticks) and rustdoc will  make it a clickable link! [I don't know, but it wouldn't surprise me if this also allows editors to make this a followable link.]

Reply:
Will fix.

Reply:
Fixed in [c45e5e6](https://github.com/ykjit/yk/pull/578/commits/c45e5e644b2bc6b43c9d456a8192a5cc3ddbaff8)